### PR TITLE
Add github actions configuration

### PR DESCRIPTION
**Pull request by kplatis**

### CODE REVIEW

**Review by lecriste — APPROVED** [review]
LGTM

**Review by lecriste — APPROVED** [review]
LGTM

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "bbp-atlas-pipeline-validator"
+version = "0.3.3"
```
Comment (lecriste) [inline pyproject.toml]: What is this `version` representing? the latest package tag?

Comment (kplatis) [reply pyproject.toml]: yes. So when the package will be built, it will take this version

Comment (lecriste) [reply pyproject.toml]: that means that one must update this value every time a new tag is created... can we use something like [this](https://github.com/BlueBrain/bbp-atlas-pipeline-validator/pull/1/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7L7) `use_scm_version=True`?

Comment (kplatis) [reply pyproject.toml]: You are right, I modified it

**Review by lecriste — APPROVED** [review]

**Pull request merged**